src/code/z_demo.c

```diff
@@ -118,12 +120,13 @@ void Cutscene_DrawDebugInfo(PlayState* play, Gfx** dlist, CutsceneContext* csCtx
     GfxPrint_SetColor(&printer, 255, 255, 255, 32);
     GfxPrint_Printf(&printer, "%06d", csCtx->frames);
     GfxPrint_SetColor(&printer, 50, 255, 255, 60);
-    GfxPrint_SetPos(&printer, 4, 26);
+    GfxPrint_SetPos(&printer, 10, 26);
```

Comment:
Was this an intentional change here?

Reply:
I think it's a leftover from an attempt to enhance a bit those features, I made a separate branch for those kind of changed tho so I can revert this, I can't remember why I changed that tho

Reply:
oh yea I remember now, I changed that because I couldn't see the ``"SKIP=(START) or (Cursole Right)"`` text on screen, so I thought maybe it was behind the black bars, reverted 👍